### PR TITLE
Move lib/repair to lib/private/repair

### PR DESCRIPTION
**Pull request by MorrisJobke**
I tested it and the repair steps still run fine.

This moves the last forgotten directory into the lib/private/ subfolder.

cc @nickvergessen @schiesbn @icewind1991 @rullzer @LukasReschke @nickvergessen @DeepDiver1975

### CODE REVIEW

**Comment by MorrisJobke** [comment]
cc @Xenopathic - makes the autoloader simpler ;)

File: tests/lib/autoloader.php [hunk 1]
```diff
@@ -90,7 +90,7 @@ public function testLoadCoreNamespaceSettings() {
 
 	public function testLoadCoreNamespaceRepair() {
 		$this->assertEquals([
-			\OC::$SERVERROOT . '/lib/repair/foo/bar.php', 
+			\OC::$SERVERROOT . '/lib/private/repair/foo/bar.php',
```
Comment (nickvergessen) [inline tests/lib/autoloader.php]: This now matches `public function testLoadCoreNamespace() {` and could therefor also be deleted

**Comment by nickvergessen** [comment]
Fine by me :)

**Comment by Xenopathic** [comment]
:+1:

File: tests/lib/autoloader.php (hunk 1 above)
Comment (MorrisJobke) [inline tests/lib/autoloader.php]: But I want to keep it in for now - just to be sure not to introduce a regression for this special case later.

**Comment by MorrisJobke** [comment]
> Fine by me :)

Is this a :+1: ?

**Comment by nickvergessen** [comment]
:+1: yes, was only waiting for a reply to my comment

**Pull request merged**